src/main/java/com/firstwebapp/practice/spring/AlbumRepository.java

```diff
@@ -6,3 +6,6 @@
 
 }
 
+
+

```

Comment:
And these extra newlines at the end of files. Not sure why these were added at all, or why they were committed to git.